content/ghc-9.14.1-alpha1/index.md

```diff
@@ -0,0 +1,65 @@
+---
+authors: ["Ben Gamari"]
+title: "GHC 9.14.1-alpha1 is now available"
+date: 2025-08-19
+categories: ["GHC"]
+tags: ["Release"]
+---
+
+The GHC developers are very pleased to announce the availability of the
+first alpha prerelease of GHC 9.14.1. Binary distributions, source
+distributions, and documentation are available at [downloads.haskell.org][].
+
+GHC 9.14 will bring a number of new features and improvements, including:
+
+* Significant improvements in specialisation:
+
+   * The `SPECIALISE` pragma now allows use of type application syntax
+
+   * The `SPECIALISE` pragma can be used to specialise for expression arguments
+     as well as type arguments.
+
+   * Specialisation is now considerably more reliable in the presence of
+     `newtype`s, adding #23109
```

Comment:
Is this a stub? In any case a direct link to the issue may be more helpful.

Reply:
Fixed. Thanks!